sbom/examples/container_image/release/from_catalog.py

```diff
@@ -32,14 +36,16 @@ def get_rpms(image_id):
     return sorted(response.json()["rpms"], key=lambda rpm: rpm["nvra"])
 
 
-def create_sbom(image_id, root_package, packages, rel_type):
-    relationships = [
+def create_sbom(image_id, root_package, packages, rel_type, other_pkgs=[], other_rels=[]):
```

Comment:
These should be `other_pkgs=None, other_rels=None` and then be assigned to empty lists if they are None, otherwise all editors and linters will complain about mutable default arguments.

Reply:
Thanks, didn't realise I'd gotten rusty :)